dashboard/src/components/UpgradeForm/UpgradeForm.test.tsx

```diff
@@ -96,14 +98,14 @@ const populatedProps = {
     appVersion: testVersion.appVersion,
     readme: "readme",
     readmeError: undefined,
-    values: "values",
+    values: "initial: values",
```

Comment:
The string value used for the `values` field needs to be something that can be parsed as a yaml hash or array. So I've updated it here to `initial: values`, which has also shown that these will be included in upgrade tests below.